ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/SubscriptionDetailsView.kt

```diff
@@ -56,295 +20,18 @@ internal fun SubscriptionDetailsView(
-                ),
-            )
-            Text(
-                text = subtitle,
-                style = subtitleTextStyle,
-                modifier = Modifier.alpha(
-                    if (prominentSubtitle) SettingsRowMainTextAlpha else SettingsRowSupportingTextAlpha,
-                ),
-            )
-        }
-    }
-}
-
-@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
-private fun getExpirationValue(
-    expirationOrRenewal: ExpirationOrRenewal,
-    localization: CustomerCenterConfigData.Localization,
-): String {
-    val expirationValue =
-        when (expirationOrRenewal.date) {
-            is ExpirationOrRenewal.Date.DateString -> expirationOrRenewal.date.date
-            ExpirationOrRenewal.Date.Never -> localization.commonLocalizedString(
-                CommonLocalizedString.NEVER,
-            )
-        }
-    return expirationValue
-}
-
-@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
-private fun labelForExpirationOrRenewal(
-    expirationOrRenewal: ExpirationOrRenewal,
-    localization: CustomerCenterConfigData.Localization,
-): String {
-    val stringKey = when (expirationOrRenewal.label) {
-        ExpirationOrRenewal.Label.EXPIRES -> CommonLocalizedString.EXPIRES
-        ExpirationOrRenewal.Label.EXPIRED -> CommonLocalizedString.EXPIRED
-        ExpirationOrRenewal.Label.NEXT_BILLING_DATE -> CommonLocalizedString.NEXT_BILLING_DATE
-    }
-    return localization.commonLocalizedString(stringKey)
-}
-
-@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
-private fun getPrice(
-    purchaseInformation: PurchaseInformation,
-    localization: CustomerCenterConfigData.Localization,
-): String? {
-    return when (purchaseInformation.price) {
-        PriceDetails.Free -> localization.commonLocalizedString(CommonLocalizedString.FREE)
-        is PriceDetails.Paid -> purchaseInformation.price.price
-        PriceDetails.Unknown -> null
-    }
-}
-
-@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
-private fun getSubscriptionExplanation(
-    purchaseInformation: PurchaseInformation,
-    localization: CustomerCenterConfigData.Localization,
-): String {
-    val stringKey = when (purchaseInformation.explanation) {
-        Explanation.APPLE -> CommonLocalizedString.APPLE_SUBSCRIPTION_MANAGE
-        Explanation.PROMOTIONAL -> CommonLocalizedString.YOU_HAVE_PROMO
-        Explanation.WEB -> CommonLocalizedString.WEB_SUBSCRIPTION_MANAGE
-        Explanation.OTHER_STORE_PURCHASE -> CommonLocalizedString.PLEASE_CONTACT_SUPPORT
-        Explanation.AMAZON -> CommonLocalizedString.AMAZON_SUBSCRIPTION_MANAGE
-        Explanation.EARLIEST_RENEWAL -> CommonLocalizedString.SUB_EARLIEST_RENEWAL
-        Explanation.EARLIEST_EXPIRATION -> CommonLocalizedString.SUB_EARLIEST_EXPIRATION
-        Explanation.EXPIRED -> CommonLocalizedString.SUB_EXPIRED
-        Explanation.LIFETIME -> CommonLocalizedString.YOU_HAVE_LIFETIME
+        PurchaseInformationCardView(
```

Comment:
Moved the logic to `PurchaseInformationCardView` but kept `SubscriptionDetailsView` since it's being reused for multiple subscriptions support